web/src/components/pages/createGamePage.tsx

```diff
@@ -2,12 +2,14 @@ import * as React from 'react';
 import RedirectToLoginIfNotLoggedIn from '../utilities/redirectToLoginIfNotLoggedIn';
 import CreateGameForm from '../forms/createGameForm';
 import BasicPageContainer from '../sections/basicPageContainer';
+import LoadAllBoards from '../utilities/loadAllBoards';
 
 export default class CreateGamePage extends React.Component<{}> {
     render() {
         return (
             <BasicPageContainer>
                 <RedirectToLoginIfNotLoggedIn/>
+                <LoadAllBoards/>
```

Comment:
This only needs to happen on first page load, and if the repective boards havent already been loaded.

Reply:
This might make sense to iterate on when dealing with other page navigation issues.

Reply:
Created issue #342 to follow up on this